tests/expected/shadow/uninit_array/test.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::alloc::{alloc, dealloc, Layout};
+
+static mut SM: kani::shadow::ShadowMem = kani::shadow::ShadowMem::new();
+
+fn write(ptr: *mut i32, offset: usize, x: i32) {
+    unsafe {
+        let p = ptr.offset(offset as isize);
+        p.write(x);
+        SM.set_init(p as *mut u8, true);
```

Comment:
For initialization, this should actually write 4 times to the shadow memory.

Since this is just a sanity check, I would just change the types you are using to `i8` so you don't have to bother.

Reply:
Indeed. I'll change it.

Reply:
Done.